sdk/purview/azure-purview-administration/README.md

```diff
@@ -0,0 +1,170 @@
+# Azure Purview Administration client library for Python
+
+Azure Purview is a fully managed cloud service.
+
+**Please rely heavily on the [service's documentation][account_product_documentation] to use this library**
+
+[Source code][source_code] |  [Package (PyPI)][account_pypi] | [API reference documentation][account_ref_docs]| [Product documentation][account_product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- Python 2.7, or 3.6 or later is required to use this package.
+- You must have an [Azure subscription][azure_subscription] and a [Purview][purview_resource] to use this package.
+
+#### Create a Purview Resource
+
+Follow [these][purview_resource] instructions to create your Purview resource
+
+### Install the package
+
+Install the Azure Purview Account client library for Python with [pip][pip]:
+
+```bash
+pip install azure-purview-administration
+```
+
+### Authenticate the client
+
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip] and
+[enable AAD authentication on your Purview resource][enable_aad]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
+As an example, [DefaultAzureCredential][default_azure_credential]
+can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+
+Use the returned token credential to authenticate the client:
+
+```python
+from azure.purview.administration.account import PurviewAccountClient
+from azure.purview.administration.metadatapolices import PurviewMetadataPoliciesClient
+from azure.identity import DefaultAzureCredential
+
+credential = DefaultAzureCredential()
+account_client = PurviewAccountClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+metadatapolicy_client = PurviewMetadataPoliciesClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+```
+
+## Key concepts
+
+### Client
+
+## Examples
+
+The following section shows you how to initialize and authenticate your client, then list all of your keys.
+
+- [Get Keys](#get-keys "Get All Keys")
+
+### Get Keys
+
+```python
+from azure.purview.administration.account import PurviewAccountClient
+from azure.identity import DefaultAzureCredential
+
+credential = DefaultAzureCredential()
+client = PurviewAccountClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+response = client.accounts.get_access_keys()
```

Comment:
This is a good point, I don't see more than one operation group indeed. This would require Swagger changes though, so let me know a little more how bad I want it NOW or later.

Reply:
I think in the interest of time, it's not 100% needed, but it does seem weird calling `PurviewAccountClient().accounts....`, because of the duplication of accounts.

Reply:
Actually there is a second operation group in Account:
https://github.com/Azure/azure-rest-api-specs/blob/4c8162b0a1f7bbd46e9aedc0e19bbe181e549c4c/specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2019-11-01-preview/account.json#L180

While I do understand, I won't suggest to rename operation groups at this point for this preview, and let's wait a GA to redicsuss this. IOW, let's keep it for now